snap/snapcraft.yaml

```diff
@@ -0,0 +1,158 @@
+name: azure-iot-identity
+base: core20 # the base snap is the execution environment for this snap
+version: '1.4-006' # just for humans, typically '1.2+git' or '1.3.2'
+summary: Provides provisioning and cryptographic services for Azure IoT Hub devices.
+description: |
+  The Identity Service provisions a device's identity and any modules it runs. The device identity can be based
+  on symmetric keys or X.509 certificates. It supports manual device registrations or individual/group
+  enrollments with the Azure Device Provisioning Service.
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict
+
+parts:
+  rust-toolchain:
+    plugin: nil
+    build-packages:
+      - curl
+    build-environment:
+      - BINDGEN_VERSION: '0.54.0'
+      - CBINDGEN_VERSION: '0.15.0'
+      - PATH: "$PATH:$HOME/.cargo/bin"
+    override-build: |
+      mkdir -p $HOME/.cargo/bin
+      curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --no-modify-path --profile minimal -y
+      cargo install bindgen --version "=$BINDGEN_VERSION"
+      cargo install cbindgen --version "=$CBINDGEN_VERSION"
+  iot-identity-services:
+    build-environment:
+      - PATH: "$PATH:$HOME/.cargo/bin"
+      - ARCH: "$SNAPCRAFT_TARGET_ARCH"
+    after: [ rust-toolchain ]
+    plugin: nil
+    source: ./
+    build-packages:
+      - acl
+      - autoconf
+      - autoconf-archive
+      - automake
+      - build-essential
+      - clang
+      - cmake
+      - curl
+      - git
+      - jq
+      - libclang1
+      - libltdl-dev
+      - libssl-dev
+      - libtool
+      - libtss2-dev
+      - llvm-dev
+      - pkg-config
+      - to arm64:
+        - ca-certificates
+        - libcurl4-openssl-dev
+    stage-packages:
+      - libtss2-esys0
+    override-build: |
+      contrib/third-party-notices.sh > THIRD-PARTY-NOTICES
+      make install-deb \
+        RELEASE=1 \
+        PLATFORM_FEATURES=snapd \
+        USER_AZIOTID=root \
+        USER_AZIOTCS=root \
+        USER_AZIOTKS=root \
+        USER_AZIOTTPM=root \
+        SOCKET_DIR=/var/sockets/aziot \
+        DESTDIR=$SNAPCRAFT_PART_INSTALL
+    organize:
+      usr/: .
+    stage:
+      - -include
+      # - -lib/systemd
+      - -var
+    override-prime: |
+      snapcraftctl prime
+      ln -vrfs $SNAPCRAFT_PRIME/libexec/aziot-identity-service/aziotd $SNAPCRAFT_PRIME/libexec/aziot-certd
+      ln -vrfs $SNAPCRAFT_PRIME/libexec/aziot-identity-service/aziotd $SNAPCRAFT_PRIME/libexec/aziot-identityd
+      ln -vrfs $SNAPCRAFT_PRIME/libexec/aziot-identity-service/aziotd $SNAPCRAFT_PRIME/libexec/aziot-keyd
+      ln -vrfs $SNAPCRAFT_PRIME/libexec/aziot-identity-service/aziotd $SNAPCRAFT_PRIME/libexec/aziot-tpmd
+  command-chain:
+    plugin: dump
+    source: ./contrib
+    stage: [ snap/command-chain ]
+
+apps:
+  aziotctl:
+    command: bin/aziotctl
```

Comment:
```suggestion
    command: usr/bin/aziotctl
```

Reply:
So the way the snap is constructed now, `bin/aziotctl` is the correct path, because I explicitly relocated `usr/bin/` to `bin/`. There's no good reason to do this except my personal preference is to keep snap directory structure as flat as possible (one bin, one lib, one shared, avoid nesting things under usr) but I should emphasize that that is my personal preference, not established best practices. So if to keep things consistent you'd like to put aziotctl under usr/bin within the snap, that would be fine with me